src/react/header/Header.tsx

```diff
@@ -0,0 +1,145 @@
+import clsx from 'clsx'
```

Comment:
I used a small package called `clsx` 
https://www.npmjs.com/package/clsx
the code is small and simple 
https://github.com/lukeed/clsx/blob/master/src/index.js